src/MultiplayerMod/Multiplayer/CoreOperations/PlayersManagement/PlayersManagementController.cs

```diff
@@ -65,10 +68,12 @@ UnityTaskScheduler scheduler
     }
 
     private void OnClientStateChanged(MultiplayerClientState state) {
-        if (state != MultiplayerClientState.Connected)
-            return;
-
-        client.Send(new InitializeClientCommand(profileProvider.GetPlayerProfile()));
+        if (state == MultiplayerClientState.Connected)
+            client.Send(new InitializeClientCommand(profileProvider.GetPlayerProfile()));
+        if (state == MultiplayerClientState.Error) {
+            events.Dispatch(new GameQuitEvent(multiplayer));
```

Comment:
Won't it be triggered on quit?

Reply:
It does not. The main purpose here to stop client and server components from triggering new network calls (e.g. sending cursor coordinates or syncing other UI actions)

Reply:
Oh, not sure if it's okay to trigger "game quit", because, well, it's not game quit.

Reply:
Okay, it will be addressed in our QoL 2, I guess, in network platform segregation.